examples/logging.rs

```diff
@@ -11,10 +11,12 @@
 //!
 //! To use the dioxus logger in your app, simply call any of the tracing functions (info!(), warn!(), error!())
 
-use dioxus::logger::tracing::{debug, error, info, warn};
+#[allow(unused_imports)]
+use dioxus::logger::tracing::{debug, error, info, warn, Level};
 use dioxus::prelude::*;
 
 fn main() {
+    dioxus::logger::init(Level::INFO).expect("Failed to initialize logger");
```

Comment:
can you put a comment here say that `dioxus::logger::init` is *optional* and called automatically by `dioxus::launch`, but you *can* call it yourself manually in the cases you:
- want to customize its behavior 
- aren't using `dioxus::launch` (ie custom fullstack setups) but want the integration.